web/client-api/client-api.gradle

```diff
@@ -1,14 +1,15 @@
 plugins {
     id 'io.deephaven.project.register'
+    id 'java-library'
 }
 
 apply from: "$rootDir/gradle/web-client.gradle"
 
 dependencies {
-    compile project(':web-shared-beans')
-    compile project(':web-client-backplane')
+    api project(':web-shared-beans')
```

Comment:
Why?

Reply:
`public class TableHandle extends ServerReplyHandle`

Reply:
Part of web-client-api public api.

Reply:
This is wrong, but only in theory - i'll continue to clean this mess up on my side.